frontend/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
```

Comment:
package changes likely got pulled in from the other PR? probably shouldn't be in here